### PR TITLE
support .less files

### PR DESCRIPTION
**Pull request by Wadimich**
Support .less files
Package renamed (JetBrains to Jetbrains)

### CODE REVIEW

File: src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 package com.almightyalpaca.jetbrains.plugins.discord;
 
-public class JetBrainsDiscordIntegration
+public class JetbrainsDiscordIntegration
```
Comment (MinnDevelopment) [inline src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java]: What does this have to do with `.less` files?

Comment (MinnDevelopment) [reply src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java]: Also, `JetBrains` is the correct spelling for the company name.

Comment (Wadimich) [reply src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java]: It should identify .less files as supported, not unknown files.

Should i return package name back to JetBrainsDiscordIntegration?

Comment (MinnDevelopment) [reply src/main/java/com/almightyalpaca/jetbrains/plugins/discord/JetbrainsDiscordIntegration.java]: If its unrelated to the support of `.less` files yes
